### DISCUSSION TITLE
Potential license issue with third party font

### DISCUSSION THREAD

**Discussion by trollixx** [hasura/graphql-engine#9525] [General]
During internal license compliance review of Hasura it was discovered that it potentially includes proprietary `ARCADECLASSIC.ttf` font. Quick googling revealed that the font is free for non-commercial use but is not available under open license. I can see that the font is being used in both CE and EE versions. Was there a special agreement made with the author so it can be used in the open source version? Thanks.

**Comment by manasag** [comment]
Hi @trollixx , thanks for raising this with us. This is not intended, and we are working to remediate the situation.

**Reply by trollixx** [reply]
@manasag any updates on this? It looks like the font is still in use. Thanks.

**Reply by jbotelho2-bb** [reply]
For reference, this font seems to be from https://www.dafont.com/arcade-classic-2.font
The font download includes the following notice:

> Thank you for downloading this font!
> 
> This font is copyright (c) Jakob Fischer at www.pizzadude.dk,  all rights reserved. Do not distribute without the author's permission.
> 
> Use this font for non-commercial use only! If you plan to use it for commercial purposes, contact me before doing so!
> 
> 
> For more original fonts take a look at www.pizzadude.dk
> 
> Have fun and enjoy!
> 
> Jakob Fischer
> jakob@pizzadude.dk
> www.pizzadude.dk
